addons/medical_engine/XEH_preInit.sqf

```diff
@@ -52,4 +58,21 @@ addMissionEventHandler ["Loaded", {
     } forEach GVAR(fixedStatics);
 }];
 
+["ace_unconscious", {
+    params ["_unit", "_active"];
+    if (_active) then {
+        if (_unit getVariable [QGVAR(waitForAnim), true]) then {
+            [{(animationstate _this) find QGVAR(face) != -1}, {
+                [_this, animationstate _this] call FUNC(applyAnimAfterRagdoll);
+            }, _unit, 20] call CBA_fnc_waitUntilAndExecute;
```

Comment:
What if client disconnects in these 20 seconds?

Reply:
animationState objNull returns `""`
find will always be -1, will run till timeout and fail.

Reply:
After 20 seconds nothing should happen.